### ISSUE TITLE
Doesn't work in iOS earlier than 7

### ISSUE THREAD

**Issue by parsley72** [PeqNP/FTPKit#15]
If I try using FTPKit in iOS5 I get a crash in [FTPClient setQueue]:
Thread 1: EXC_BAD_ACCESS (code=1, address=0x661af48d)

Has anyone else tried using the library in earlier versions of iOS when building against the 8.2 SDK?

**Comment by ReDetection** [comment]
Yes, I did, iOS 6 works well — and I made it required platform in the podspec. 
You're getting error because in the versions of iOS prior to 6 `dispatch_queue_t` was not an object yet — just a struct. So if you really wanna use thus on iOS 5, you should consider changing `strong` property type to `assign`. http://stackoverflow.com/questions/8904206/what-property-should-i-use-for-a-dispatch-queue-after-arc
